packages/jaeger-ui/src/components/Monitor/ServicesView/operationDetailsTable/index.tsx

```diff
@@ -235,12 +242,18 @@ export class OperationTableDetails extends React.PureComponent<TProps, TState> {
               },
             };
           }}
-          onChange={(pagination, filters, { columnKey, order }) => {
-            if (!isEqual({ columnKey, order }, this.state.tableSorting)) {
-              const clickedColumn = tableTitles.get(columnKey || this.state.tableSorting.columnKey);
+          onChange={(pagination, filters, sorter) => {
```

Comment:
what specifically is changing in antd that requires these modifications, which look more complicated than before?

Reply:
In antd 4x, sorting by multiple sortable columns now reports all columns that are active in sorting, so the event handling code must also accept an array.